ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/openshift-e2e-test-hypershift-qe-mgmt-chain.yaml

```
@@ -5,6 +5,11 @@ chain:
   - ref: idp-htpasswd
   - ref: openshift-extended-test
   - ref: openshift-e2e-test-qe-report
+  env:
+    - name: TEST_FILTERS
+      default: "~ChkUpgrade&;~NonPreRelease&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&"
```

```suggestion
      default: "~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;HyperShiftMGMT&;~MicroShiftOnly&"
```
It should also excluded the Serial and disruptive test cases for non long during tests,right?

---

no, long duration maybe a disruptive case, especially, some apiserver cases are disruptive and longduration.

---

Got it, so for hyppershift mgmt jobs do not separate disruptive/non-disruptive jobs.